__snapshots__/manifest.js

```diff
@@ -163,3 +163,7 @@ For a better experience, it is recommended to use either rebase-merge or squash-
 
 _More technical details can be found at [stainless-api/release-please](https://github.com/stainless-api/release-please)_.
 `
+
+exports['Manifest createReleases should throw when release branch is missing and changes-branch not in synced with target-branch 1'] = `
+Branch 'next' cannot be safely re-aligned with 'main', and will likely result in git conflicts when the next release PR is created. Hint: compare release-please--branches--main--changes--next, next, and main for inconsistencies
+`
```

Comment:
@RobertCraigie Interested to get your feed on this error message. If you see this in a Vercel log, is that helpful to debug the problem?